File: emall-parent/manager-web/src/main/webapp/js/services/brandservice.js
//app模块的服务层
app.service('brandService', function($http) {
	this.findPage = function(page, size, serachEntity) {
		return $http.post('../brand/querypager?page=' + page + '&size=' + size,
				serachEntity);
	}

	this.update = function(entity) {
		return $http.post('../brand/update', entity);
	}

	this.save = function(entity) {
		return $http.post('../brand/add', entity);
	}

	this.queryid = function(id) {
		return $http.get('../brand/queryid?id=' + id);
	}

	this.deletes = function(ids) {
		return $http.get('../brand/delete?ids=' + ids);
	}
})

File: emall-parent/seller-api/src/main/java/cn/li/seller/service/BrandService.java
package cn.li.seller.service;

import java.util.List;

import cn.li.emall.entity.TbBrand;
import cn.li.ultil.ShopResult;

/**
 * 品牌接口
 *
 */
public interface BrandService {
	
	public List<TbBrand> findAll();
	
	public ShopResult querypager(TbBrand tbbrand,int page,int size);
	
	public void add(TbBrand tbbrand);
	
	public TbBrand queryid(long id);
	
	public void update(TbBrand tbbrand);
	
	public void delete(long[] ids);
}
